File: push-openapi-php-demo/BindTag.php
<?php

include_once '../aliyun-php-sdk-core/Config.php';
use \Push\Request\V20150827 as Push;

$accessKeyId = "";
$accessSecret = "";
$appKey = 123456;
$deviceId = '';
$account = '';
$iClientProfile = DefaultProfile::getProfile("cn-hangzhou", $accessKeyId, $accessSecret);
$client = new DefaultAcsClient($iClientProfile);
$request = new Push\BindTagRequest();

$request->setAppKey($appKey);
$request->setKeyType(1);//1: device 2: account
$request->setClientKey($deviceId);
$request->setTagName("tag2");

$response = $client->getAcsResponse($request);
print_r("\r\n");
print_r($response);

?>

File: push-openapi-net-demo/BindTag.cs
using Aliyun.Acs.Core;
using Aliyun.Acs.Core.Exceptions;
using Aliyun.Acs.Core.Profile;
using Aliyun.Acs.Push.Model.V20150827;
using System;

namespace AlibabaCloud
{
    class BindTag
    {
        static void Main()
        {
            IClientProfile clientProfile = DefaultProfile.GetProfile("cn-hangzhou", "<your access key id>", "<your access key secret>");
            DefaultAcsClient client = new DefaultAcsClient(clientProfile);
            BindTagRequest request = new BindTagRequest();
            request.AppKey = <Your AppKay>;
            request.KeyType = 1;//1 : device 2 : account
            request.ClientKey = <Your DeviceId>;
            request.TagName = "tag1";

            try
            {
                BindTagResponse response = client.GetAcsResponse(request);
                Console.WriteLine("RequestId:" + response.RequestId);
                Console.ReadLine();
            }
            catch (ServerException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
                Console.ReadLine();
            }
            catch (ClientException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
                Console.ReadLine();
            }
        }

    }
}
